front/app/components/Form/tracks.ts

```diff
@@ -0,0 +1,4 @@
+export default {
+  ideaFormSubmitted: 'Idea form submitted',
+  surveyFormSubmitted: 'Survey form submitted',
+};
```

Comment:
Just curious about the intended use case/analysis here - is it to analyze the # of submissions? Or something else? :)

Reply:
Making a funnel that tracks a user from visiting the project page to submitting their idea. It looks like this: 
![image](https://github.com/user-attachments/assets/30c38e4e-2c41-4f1c-b117-e7bfef8f8bf1)

Reply:
Gotcha! Thanks! :)